plugins/jake-node/jake-node.plugin.zsh

```diff
@@ -6,8 +6,8 @@
 # Inspiration : https://weblog.rubyonrails.org/2006/3/9/fast-rake-task-completion-for-zsh
 
 function _jake () {
-  if [ -f Jakefile ]||[ -f jakefile ]; then
-    compadd `jake -T | cut -d " " -f 2 | sed -E "s/.\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]//g"`
+  if test -f Jakefile || test -f Jakefile.js || test -f jakefile || test -f Jakefile.js; then
```

Comment:
`Jakefile.js` is repeated. Could you also keep the `[[ -f ]]` syntax?
```suggestion
  if [[ -f Jakefile || -f Jakefile.js || -f jakefile || -f jakefile.js ]]; then
```

Reply:
@mcornella

> Jakefile.js is repeated.

My bad, fixed.

> Could you also keep the [[ -f ]] syntax?

Yes, I created new PR.